tiatoolbox/models/dataset/classification.py

```diff
@@ -132,31 +123,24 @@ def __getitem__(self, idx):
 
 class WSIPatchDataset(abc.PatchDatasetABC):
     """Defines a WSI-level patch dataset.
-
     Attributes:
         reader (:class:`.WSIReader`): an WSI Reader or Virtual Reader
         for reading pyramidal image or large tile in pyramidal way.
-
         inputs: List of coordinates to read from the `reader`,
         each coordinate is of the form [start_x, start_y, end_x, end_y].
-
         patch_size: a tuple(int, int) or ndarray of shape (2,).
         Expected size to read from `reader` at requested `resolution` and `units`.
         Expected to be (height, width).
-
         lv0_patch_size: a tuple (int, int) or ndarray of shape (2,).
         `patch_size` at level 0 in `reader` at requested `resolution`
         and `units`. Expected to be (height, width).
-
         resolution: check (:class:`.WSIReader`) for details.
         units: check (:class:`.WSIReader`) for details.
-
         preproc_func: Preprocessing function used to transform the input data. If
         supplied, then torch.Compose will be used on the input preprocs.
         preprocs is a list of torchvision transforms for preprocessing the image.
         The transforms will be applied in the order that they are given in the list.
         https://pytorch.org/vision/stable/transforms.html.
-
```

Comment:
why deleting these spaces?

Reply:
changing indentation for readability. Re: space, wsireader doesnt have space between arg doc spanning multiline so i match that. I think a space would make it easier to read though.

Reply:
As reply above, to match with wsireader format form.